io.jor.bugdom.yml

```diff
@@ -0,0 +1,38 @@
+app-id: io.jor.bugdom
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: Bugdom
+rename-desktop-file: bugdom.desktop
+rename-icon: bugdom-desktopicon
+
+modules:
+  - name: bugdom
+    buildsystem: cmake
+    sources:
+      - type: git 
+        url: https://github.com/jorio/Bugdom.git
+        tag: 1.3.1
+        commit: d4b5e729cf6747f1b07ee2752c824ac882f5ce5b
+      - type: file
+        path: bugdom_launcher.sh
+      - type: file
+        path: io.jor.bugdom.metainfo.xml
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    no-make-install: true
+    build-commands:
+      - install -D bugdom_launcher.sh ${FLATPAK_DEST}/bin/Bugdom
+      - mkdir -p ${FLATPAK_DEST}/bugdom
+      - install -D ${FLATPAK_BUILDER_BUILDDIR}/Bugdom ${FLATPAK_DEST}/bugdom/Bugdom
+      - cp -r ./Data ${FLATPAK_DEST}/bugdom
+      - install -D ./packaging/bugdom.desktop ${FLATPAK_DEST}/share/applications/bugdom.desktop
+      - install -D ./packaging/bugdom-desktopicon.png ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/bugdom-desktopicon.png
+      - install -D ./io.jor.bugdom.metainfo.xml ${FLATPAK_DEST}/share/metainfo/io.jor.bugdom.metainfo.xml
+
+finish-args:
```

Comment:
Nitpick: by convention, `finish-args` usually is before `modules`.

Reply:
it's also mssing --share=ipc